datapower-docker-compose-dp-mq/datapower-docker-compose-dp-mq.mkd

```diff
@@ -0,0 +1,168 @@
+## A Composed Sample Application with DataPower and MQ 
+This sample demonstrates how to utilize multiple containers, orchestrated by Docker Compose, to create an environment that includes DataPower receiving and processing a request and writing the resulting message to an MQ back end. The composed application includes everything you need -- the load driver, the DataPower, and the MQ back-end. Everything is there from config, to stylesheets, to docker-compose.yml. In this sample, I'll outline the travels of the request message, `bob`, through the docker-composed application.
+
+This is a highly customizable example and it is encouraged that you play around with it.
+
+## Prerequisites
+Before you attempt this tutorial, please be sure that you:
+
+- Read, and better yet attempt, the [Docker Compose Hello World Application](https://github.com/ibm-datapower/datapower-tutorials/blob/master/datapower-docker-compose-hello-world/datapower-docker-compose-hello-world.md) sample. This is a great starting point that offers some of the detailed framework of how DataPower containers can be created and used in a continuous integration system.
+- For your convenience, I created and used a Docker MQ base image. If you wish to run this step manually, please follow the instrucitons on [ibmcom/mq](https://hub.docker.com/r/ibmcom/mq/) on Docker Hub to create your own base Docker MQ image.
+
+
+## The Composed Application
+No rock band here, just the message Bob. Harley's Hello World example (mentioned in the Prerequisite section) goes into great detail about the various elements of his project. In order to avoid redundancy, I will not go into such detail here, but I will still cover the whole project. Let's take a look at the `docker-compose.yml` file that orchestrates the entire environment.
```

Comment:
I'm not opposed to "Bob" - I even think it's handy to name the relevant message you want people to follow. The only thing I would suggest then is to be super careful and consistent about using "Bob" and "the message" since that might imply we are now talking about different things.

See the section after you explain the pieces in the application (no mention of bob here)  and the last section of "Meet Bob" where you indicate a datapower service clears the queue and throws "the message" away.